go/cmd/internal/docgen/docgen.go

```diff
@@ -190,11 +196,14 @@ func anonymizeHomedir(file string) (err error) {
 	if err != nil {
 		return err
 	}
+	if _, err := os.Stat(file); err != nil {
+		return nil
+	}
 
 	// We're replacing the stuff inside the square brackets in the example sed
 	// below:
 	// 	's:Paths to search for config files in. (default \[.*\])$:Paths to search for config files in. (default \[<WORKDIR>\]):'
-	sed := exec.Command("sed", "-i", "-e", fmt.Sprintf("s:%s:<WORKDIR>:i", wd), file)
+	sed := exec.Command("sed", "-i", "", "-e", fmt.Sprintf("s:%s:%s:", wd, "<WORKDIR>"), file)
```

Comment:
Looks like this has always been failing because the `sed` command was incorrect. Without this change the `-e` is seen as the value for `-i` and the `sed` fails.